Dockerfile.ocp

```diff
@@ -9,9 +9,8 @@ RUN prepare-image.sh && \
   rm -f /bin/prepare-image.sh
 
 COPY ironic-inspector.conf.j2 /etc/ironic-inspector/
-COPY scripts/ /bin/
+COPY scripts/* /bin/
```

Comment:
Why are we not doing this upstream first?

Reply:
because upstream works fine as it is, this is for downstream only and it currently breaks the ironic-inspector builds
I can do that upstream too if we want to keep it in sync, but it's not really needed

Reply:
OK, seems like it will be easier long term if this change is upstream as well, but no reason to block.

Reply:
thanks, I totally agree, I'll change that upstream as well, just for consistency